viewmodel-events/common/src/main/kotlin/Event.kt

```diff
@@ -0,0 +1,4 @@
+package com.redmadrobot.gears.viewmodelevents
+
+/** Marker interface for entities that can be put to the ViewModelEvents. */
+public interface Event
```

Comment:
Should we also rename this interface?

Reply:
I'd suggest smth like ViewModelEvent?

Reply:
The only problem with this variant is that it differs from `ViewModelEvents` only by one character, and it is easy to make a mistake.

Reply:
Maybe SingleEvent then? Or just leave this naming as it is